src/DETHRACE/common/raycast.c

```diff
@@ -253,9 +243,9 @@ int DRScenePick2D(br_actor* world, br_actor* camera, dr_pick2d_cbfn* callback, v
     DRActorToRoot(camera, world, &camera_tfm);
     BrMatrix34Inverse(&gPick_model_to_view__raycast, &camera_tfm);
     scale = cos(BrAngleToRadian(camera_data->field_of_view / 2));
-    scale = scale / sin(scale);
+    scale = scale / sin(BrAngleToRadian(camera_data->field_of_view / 2));
```

Comment:
this change doesn't look correct based on my looking of the OG game?

Reply:
hmm, nevermind, looking at the assembly you are right, my tooling shows the wrong thing!